src/Libraries/DSOffice/Excel.cs

```diff
@@ -270,6 +270,7 @@ public static WorkBook SaveAsExcelWorkbook(WorkBook workbook, string filename)
         /// <param name="readAsStrings">toggle to switch between reading Excel file as strings only or not</param>
         /// <returns name="data">Rows of data from the Excel worksheet.</returns>
         /// <search>office,excel,spreadsheet,ifequalreturnindex</search>
+        [IsVisibleInDynamoLibrary(false)]
```

Comment:
Hi @sharadkjaiswal , since the Excel and CSV methods will be moved to DSOffice namespace as suggested by @aparajit-pratap , I think these should be included since the DLL will be loaded and these methods should be hidden from being called from CBN.